explainer/index.bs

```diff
@@ -727,6 +727,81 @@ When using advanced methods to transfer data to the GPU (with a rolling list of
 
 ## Canvas Output ## {#canvas-output}
 
+Historically, drawing APIs (2d canvas, WebGL) are initialized from canvases using `getContext()`.
+However, WebGPU is more than a drawing API, and many applications do not need a canvas.
+WebGPU is initialized without a canvas - see [[#initialization]].
+
+Following this, WebGPU has no "default" drawing buffer.
+Instead, a WebGPU device may be connected to *arbitrarily many* canvases (via
+`GPUCanvasContext`s) and render to all of them.
+
+In order to access a canvas, an app gets a `GPUTexture` from the `GPUCanvasContext`
+and then writes to it, as it would with a normal `GPUTexture`.
+
+### Swap Chains ### {#canvas-output-swap-chains}
+
+Canvas `GPUTexture`s are vended in a very structured way:
+
+- `canvas.getContext('gpupresent')` provides a `GPUCanvasContext`.
+- `GPUCanvasContext.configureSwapChain({ device, format, usage })` provides a `GPUSwapChain`,
+    invalidating any previous swapchains, attaching the canvas to the provided device, and
+    setting the `GPUTextureFormat` and `GPUTextureUsage` for vended textures.
+- `GPUSwapChain.getCurrentTexture()` provides a `GPUTexture`.
+
+This structure provides maximal compatibility with optimized paths in native graphics APIs.
```

Comment:
nit: Maybe mention that it strongly decouples the <canvas> from the GPUDevice and allows for example swapping devices on a canvas.